pending/15-whats-a-quarren.pod

```diff
@@ -0,0 +1,226 @@
+from a single-page static, to big multi-page statics, to data-driven dynamic content. For
+each, it seems, you need a different sort of CMS.
+
+That irritated me, but for the longest time, I knew I didn't have the tools I needed to do
+the job I wanted, or the time to flesh it out. My current job gave me most all of the former;
+we use L<Dancer2> for all of our sites, with L<DBIx::Class> under the hood talking to databases.
+
+Quarren was born during the early days of the 2020 pandemic; my husband and I couldn't get
+out much, and I had some free time to scratch down some ideas--and the name?  Well, this
+CMS was born in the Quarren-tine, of course.
+
+=head2 Whence Quarren?
+
+Most of the very popular CMSes out there have one or more problems that make it unsuitable
+for all of the use cases I have. Either they are too big a hammer (a static site with Joomla?),
+too small a hammer (more than very simple CSV-based data on Jekyll) or the wrong hammer 
+altogether (try setting up search on a Hugo site. No, really, I'll wait.)
+
+That's not to say that you can't shoehorn some of the use cases into a functional
+site. You can, for instance, set up a static site in WordPress, but then you have all
+kinds of lovely features that you don't need right at hand, which is a bit of a distraction.
+I want a CMS where you have the features you need for the site you're working on close to hand, but
+none of the features that you don't need.
+
+Now, lest you think I'm knocking WordPress, I'm not; it's a fine system, as are Joomla, Jekyll, 
+Hugo, Drupal, and a host of other systems. But they're not the hammer I wanted for the nails I need
+to drive; in the case of WordPress, one of the side effects of its plugin ecosystem is that there
+are a lot of really skeevy operators out there writing plugins for pay. If you're lucky, you'll get one
+that merely spams your email box after you by their plugin. If your luck isn't all that good,
+well. It's an ugly world out there, and I didn't want to play their games.
+
+=head2 Whereby Quarren?
+
+My goals for the project are
+
+=over
+
+=item * One CMS for everything I run, instead of the current three.
+
+=item * One copy of the code running per OS install.
+
+=item * I don't want to deal with skeevy paid-plugin shops.
+
+=item * Everything available on a relatively-easy-to-use deployment ecosystem (CPAN/apt/yum)
+
+=item * Mostly DB-agnostic--from sqlite up to Oracle, if possible.
+
+=item * Mostly search-tool agnostic--deploy what works for you, or don't deploy one at all.
+
+=item * I don't want to see features I'm not using in the admin UI
+
+=item * Code I can be proud to give back to the community I love
+
+=back
+
+=head3 The "Core" of Quarren
+
+The core of the system includes pages, users, and the plugin infrastructure to find and utilize plugins.
+Literally everthing else is a plugin. A default renderer and theme is provided, to "pass through" raw HTML
```

Comment:
s/everthing/everything/

Reply:
Good catch, @racke !  I've made the change.